packages/format-po-gettext/README.md

```diff
@@ -0,0 +1,78 @@
+[![License][badge-license]][license]
+[![Version][badge-version]][package]
+[![Downloads][badge-downloads]][package]
+
+# @lingui/format-po-gettext
+
+> Read and write message catalogs in Gettext PO format with gettext-style plurals
+
+`@lingui/format-po-gettext` is part of [LinguiJS][linguijs]. See the
+[documentation][documentation] for all information, tutorials and examples.
+
+## Catalog example
+
+```po
+#. js-lingui-id: WGI12K
+#. js-lingui:icu=%7BanotherCount%2C+plural%2C+one+%7BSingular+case%7D+other+%7BCase+number+%7BanotherCount%7D%7D%7D&pluralize_on=anotherCount
+msgid "Singular case"
+msgid_plural "Case number {anotherCount}"
+msgstr[0] "Singular case"
+msgstr[1] "Case number {anotherCount}"
+```
+
+## Installation
+
+```sh
+npm install --save-dev @lingui/format-po-gettext
+# yarn add --dev @lingui/format-po-gettext
+```
+
+## Usage
+
+```js
+// lingui.config.{js,ts}
+import {formatter} from "@lingui/format-po-gettext"
+
+export default {
+  [...]
+  format: formatter({lineNumbers: false}),
+}
+```
+
+Possible options:
+
+```ts
+export type PoGettextFormatterOptions = {
+  /**
+   * Print places where message is used
+   *
+   * @default true
+   */
+  origins?: boolean
+
+  /**
+   * Print line numbers in origins
+   *
+   * @default true
+   */
+  lineNumbers?: boolean
+  
+  disableSelectWarning?: boolean
```

Comment:
any comment on this option?

Reply:
i hope no one use this shitty formatter, don't want to spend too much time on that

Reply:
To explain this option user should understand what exactly this formatter is doing and what restrictions it has. then it become obvious what exact warning this option is disabling. But it wasn't written before my refactoring and i'm not going to spend my time to adding docs for the features i'm not interested in. 
Does that make sense?